internal/cmd/workspaces.go

```diff
@@ -752,3 +754,71 @@ func buildUpdateReq(ctx context.Context, client coder.Client, conf updateConf) (
 	}
 	return &updateReq, nil
 }
+
+func openWorkspaceCmd() *cobra.Command {
+	var (
+		workspaceName string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Short:   "Open Coder workspace IDE",
+		Long:    "Open Coder workspace IDE in your default browser",
+		Example: "coder workspace open --name dev",
```

Comment:
Should name be an argument instead of a flag? Maybe both?

Sounds great to do `coder workspace open dev`.